tests/conftest.py

```diff
@@ -149,6 +149,67 @@ def global_execution_env():
         yield f, assets, n
 
 
+@pytest.fixture
```

Comment:
I think it would be good to execute this feature only once. It could be good to change its scope?

Reply:
Yes indeed, I'll do that!